corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -1,6 +1,14 @@
-/* globals django, hqDefine, hqImport */
-hqDefine('data_dictionary/js/data_dictionary', function () {
-
+/* globals define */
+define('data_dictionary/js/data_dictionary', [
```

Comment:
Do we have to duplicate the file path here? What happens if it gets out of sync with the real file path? As mentioned in another comment, see [requirejs note about named modules](http://requirejs.org/docs/api.html#modulename).

Reply:
https://github.com/dimagi/commcare-hq/pull/17824/commits/ec6c96b17824972252abd8f2d15f928d5265d0bc